{
  "_annotation": {
    "cve_id": "CVE-2024-26139",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-26139.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:citeum:opencti:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "B1287A0C-9288-524B-9981-02C756DB8985",
                "versionEndIncluding": "5.12.31",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}